### DISCUSSION TITLE
netherite-fire-resistance should have an option to include Elytras

### DISCUSSION THREAD

**Discussion by lolGavn** [PurpurMC/Purpur#913] [Feature Requests]
instead of it only being for full netherite, it should have an option for full netherite+elytra, maybe even turtle helmet too

**Comment by BillyGalbreath** [comment]
I disagree. Elytras are from phantom membranes, and are paper thin and extremely flammable. IMHO, elytras should burn just by seeing the light of day the way phantoms do ;)
